buildpacks/ruby/src/layers/bundle_install_layer.rs

```diff
@@ -321,6 +323,7 @@ fn display_name(cmd: &mut Command, env: &Env) -> String {
 }
 
 #[derive(Deserialize, Serialize, Debug, Clone, Eq, PartialEq, Default)]
+#[serde(deny_unknown_fields)]
```

Comment:
Not a metadata struct but struct implements Deserialize, so I added `#[serde(deny_unknown_fields)]`.  There were a few more examples like this.  Let me know if this needs to be reverted.

Reply:
I'm not sure that specific struct is used. Which is odd because clippy usually notices and warns on stuff like that. Having this won't block a merge.